packages/rest-client/src/types/AddonParams.ts

```diff
@@ -0,0 +1,26 @@
+import { AddonName } from './AddonName'
+
+export type AddonUcClamavVirusScanParams = {
+  purgeInfected: boolean
+}
+
+export type AddonAwsRekognitionDetectLabelsParams = undefined
+
+export type AddonRemoveBgParams = {
+  crop?: boolean
+  cropMargin?: string
+  scale?: string
+  addShadow?: boolean
+  typeLevel?: 'none' | '1' | '2' | 'latest'
+  type?: 'auto' | 'person' | 'product' | 'car'
+  semitransparency?: boolean
```

Comment:
semitransparency or semiTransparency?

Reply:
It's called `semitransparency` in REST API - https://uploadcare.com/api-refs/rest-api/v0.7.0/#operation/removeBgExecute

But that reminds me that I forgot about the camel -> kebab conversion 🤦

Reply:
I think it's better to use snake case instead of camel for addon params